src/methods.ts

```diff
@@ -10,6 +10,7 @@ export interface MethodArgument extends TypeLike {}
 
 export interface RESTMethodArgument extends MethodArgument {
     transport?: string;
+    argument?: string;
```

Comment:
Where do you need this specifically? Can't remember all details but I know we do handle annotations and their names elsewhere?

Reply:
I've done this also https://github.com/kapetacom/ui-web-components/pull/165 but I'm a little unsure if there is a better way

Reply:
Nope you're probably right - can't remember which one we use when generating code though